packages/terra-dynamic-grid/src/DynamicGrid.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { StyleSheet, css } from 'aphrodite';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import generateStyles from './styles';
+import Region from './Region';
+
+const propTypes = {
+  /**
+  * The child Region components that make up the grid.
+  */
+  children: PropTypes.node,
+  /**
+  * The layout configuration that defines the grid. Multiple layouts can be passed per mediaquery to make the grid responsive.
+  */
+  layouts: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
Yeah... I wanted to discuss changing our doc structure to more match [grommet](https://grommet.github.io/docs/notification). This would allow us to better represent complex properties better than a table can